cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ListenersValidator.java

```diff
@@ -92,6 +92,12 @@ public static void validate(int replicas, List<GenericKafkaListener> listeners)
                         validateBrokerDnsAnnotations(errors, listener, broker);
                     }
                 }
+
+                if (listener.getConfiguration().getBrokerCertChainAndKey() != null
+                    && (listener.getConfiguration().getBrokerCertChainAndKey().getSecretName() == null
+                    || listener.getConfiguration().getBrokerCertChainAndKey().getSecretName().isEmpty())) {
+                    errors.add("listener '" + listener.getName() + "' cannot have empty secret name in the brokerCertChainAndKey");
+                }
```

Comment:
Looks good. just one quick idea ... it might read better if you move it into separate method like the other validation. Plus ... would it make sense to validate the other fields as well if we already access the object?